utils/imodels-client-test-utils/src/test-context-providers/imodel/TestIModelCreator.ts

```diff
@@ -58,8 +58,7 @@ export class TestIModelCreator {
   public async createEmptyAndUploadChangesets(iModelName: string): Promise<IModelMetadata> {
     const iModel = await this.createEmpty(iModelName);
     const briefcase = await this.acquireBriefcase(iModel.id);
-    const changesetGroups = await this.createChangesetGroups(iModel.id);
```

Comment:
why was this deleted?

Reply:
I added it with changeset groups, but it's not required for any tests that use the created iModel. Therefore, it should save a few (useless) calls to API for integration tests. In my opinion, we can add it when we need it.